core/sql/executor/OrcFileReader.cpp

```diff
@@ -111,9 +133,15 @@ OFR_RetCode OrcFileReader::init()
 //    JavaMethods_[JM_FETCHBUFF2].jm_signature = "(II)[Ljava/lang/String;";
     JavaMethods_[JM_CLOSE     ].jm_name      = "close";
     JavaMethods_[JM_CLOSE     ].jm_signature = "()Ljava/lang/String;";
-   
-    return (OFR_RetCode)JavaObjectInterface::init(className, javaClass_, JavaMethods_, (Int32)JM_LAST, FALSE);
+    
+    lv_retcode = (OFR_RetCode)JavaObjectInterface::init(className,
+							javaClass_,
+							JavaMethods_,
+							(Int32)JM_LAST, javaMethodsInitialized_);
+    javaMethodsInitialized_ = TRUE;
```

Comment:
Nit. This should be "true" instead of "TRUE" since this is a bool instead of an NABoolean. But it's an unobservable bug.

Reply:
TRUE is defined in the standard include file glib.h as

#define	FALSE	(0)
#define	TRUE	(!FALSE)

I would think it should be ok to initialize bool datatype as TRUE.

Reply:
That's fine.